test/PhpSessionPersistenceTest.php

```diff
@@ -422,4 +422,30 @@ public function testPersistSessionReturnsExpectedResponseWithoutAddedCacheHeader
 
         $this->restoreOriginalSessionIniSettings($ini);
     }
+
+    public function testCookiesNotSetWithoutRegenerate(): void
```

Comment:
Usually we are not adding `: void` in tests, but if we keep it here, there should be one space before `:` (consistency).